csup/metadata.go

```diff
@@ -150,6 +150,22 @@ func (f *Float) Len(*Context) uint32 {
 	return f.Count
 }
 
+type Bytes struct {
+	Typ      super.Type `super:"Type"`
+	Bytes    Segment
+	Offsets  Segment
+	Min, Max []byte
```

Comment:
Nit: Do these like the other types in here.
```suggestion
	Min []byte
	Max []byte
```